Documentation updates (#40)
uab-cgds-worthey/rosalution: README.md

|user04 | Pre-clinical-intake user
|user07 | Bioinformatics-section user

![Rosalution logins](./docs/figures/rosalution-logins.png "Rosalution logins")

### Adding Cases

Users can create new cases on Rosalution by clicking on the '+' card to add a new case, as shown in the below figures.

> Doing this will prompt a dialog to import files to create a new Rosalution case. These files must be in .json format.

Reviewer: ```suggestion
Doing so prompts an upload dialog to import files to create a new Rosalution case. These files must be in .json format.
```

> Once uploaded, a 'Successful import' dialog will be displayed.

Reviewer: ```suggestion
Once uploaded, a 'Successful import' dialog displays.
```

Author: Thanks for pointing this out! Added suggestion.

 compose file to use, in this case `docker-compose.local-production.yml` and `--build` flag is used to force Docker
 Compose to rebuild the images.

> #### Accessing Deployed Rosalution

Reviewer: ```suggestion
#### Accessing A Locally Deployed Rosalution
```

> To access the Rosalution site after local deployment is run, open the following URL in your browser[http://local.rosalution.cgds/rosalution/](http://local.rosalution.cgds/rosalution/)

Reviewer: ```suggestion
To access the locally deployed Rosalution application after running `docker compose up`, visit the [http://local.rosalution.cgds/rosalution/](http://local.rosalution.cgds/rosalution/) URL in your browser. 
```
